spec/support/router_helpers.rb

```diff
@@ -3,6 +3,14 @@
 module RouterHelpers
   include GdsApi::TestHelpers::Router
 
+  def get_content(content_item)
+    get "/content#{content_item.base_path}"
+  end
+
+  def get_api_content(content_item)
+    get "/api/content#{content_item.base_path}"
+  end
```

Comment:
I don't think these belong in this helper.  It's all related to the router, and registering routes there, whereas these helpers are related to making requests to the content-store.